src/Microsoft.ML.CpuMath/CpuMathUtils.netcoreapp.cs

```diff
@@ -9,6 +9,7 @@
 
 namespace Microsoft.ML.Internal.CpuMath
 {
+    [BestFriend]
```

Comment:
Why is this change necessary?

Reply:
Hi Eric! Access to CpuMathUtils from FactorizationMachineInterface gives an error when using netcoreapp3.0. Issue #3654